class/defaults.yml

```diff
@@ -55,3 +55,12 @@ parameters:
           maxPods: 110
 
     debugNamespace: syn-debug-nodes
+
+    monitoring:
+      enabled: false
+      instance: ''
```

Comment:
Do we prefer "optional" parameters to be present in `class/defaults.yml` with a sentinel value, or do we prefer to not set a default at all?

From what I remember, we mostly have sentinel values in other components.

Reply:
I'm fine with both. Null could be a bit clearer though I think.